install-and-configure/advanced-configuration/parallel.md

```diff
@@ -0,0 +1,37 @@
+# Parallel Installation
+
+The below guide is optional, as there is low risk to data loss in Kubecost as all metrics are stored in cloud object-storage. The primary benefit to a parallel installation is to reduce the amount of time the Kubecost UI is unavailable/incomplete during an upgrade.
+
+It is possible to run multiple Kubecost Primaries on the same cluster in parallel. This is useful if you are upgrading to a new version and want to validate the new version before removing the current one.
+
+The primary concern is to ensure that two Kubecost cost-analyzer pods on a single cluster are not writing to the same bucket at the same time.
+
+To prevent this, add the following to the existing `values.yaml` file:
+
+```yaml
+federatedETL:
+  readOnlyPrimary: true
+```
+
+All other core settings should be the same as the existingprimary installation.
```

Comment:
I would add a link to somewhere to help define what you mean by 'existing primary installation'